tests/helpers.py

```diff
@@ -207,9 +214,16 @@ def create_pvc(sc_name, pvc_name=None):
     Returns:
         OCS: An OCS instance for the PVC
     """
+    if 'rbd' in sc_name:
+        component = constants.RBD_COMPONENT
+    elif 'cephfs' in sc_name:
+        component = constants.CEPHFS_COMPONENT
+    else:
+        component = None
+
     pvc_data = defaults.CSI_PVC_DICT.copy()
     pvc_data['metadata']['name'] = pvc_name if pvc_name else create_unique_resource_name(
-        'test', 'pvc'
+        f'test-{component}', 'pvc'
```

Comment:
```
        f'test-{component}', 'pvc'
```
It is possible that 'component' can be NoneType here. It will give 'test-None'.

Reply:
This is with reference to line 222, in case sc_name does not contain sub string 'rbd' or 'cephfs'

Reply:
jilju while creating storage with helpers.py it will have rbd or CephFS. But if someone creates storageclass explicitly I.e without helpers.py then componet will be none or later on future if we have support for rgw and till then this code is not been fixed for rgw for that componet will be none